src/meshdb/settings.py

```diff
@@ -213,3 +215,38 @@
 }
 
 HOOK_CUSTOM_MODEL = "meshapi_hooks.CelerySerializerHook"
+
+SPECTACULAR_SETTINGS = {
+    "TITLE": "MeshDB Data API",
+    "VERSION": "1.0.0",
+    "SERVE_INCLUDE_SCHEMA": False,
+    "TAGS": [
+        {"name": "API Status", "description": "Meta-endpoint to indicate API status"},
+        {"name": "Members", "description": "Members of the mesh and their contact details"},
+        {"name": "Buildings", "description": "Buildings on the mesh and their location data"},
+        {
+            "name": "Installs",
+            "description": "Installs, one corresponding to each household on the mesh or wishing to join",
+        },
+        {"name": "Links", "description": "Links between buildings"},
+        {"name": "Sectors", "description": 'Special antennas with broad coverage of a radial "slice" of land area'},
+        {"name": "Geographic & KML Data", "description": "Endpoints for geographic and KML data export"},
+        {
+            "name": "Website Map Data",
+            "description": "Endpoints used to power the nycmesh.net website map. "
+            "Uses a legacy data format, not recommended for new applications",
+        },
+        {
+            "name": "Legacy Query Form",
+            "description": "Endpoints used to power the legacy docs query form. "
+            "Uses a legacy data format, not recommended for new applications",
```

Comment:
Ditto

Reply:
I don't know that caps screaming at folks makes as much sense here. We didn't maintain schema compatibility with the existing endpoints (since we don't render CSVs). If we didn't want anyone to integrate against this, why did we build it?

Reply:
Well, I see the Query Form that I built as a hack, that's all. I would like to re-think the way that all of our current tools work, and I want people to build something that uses the "base" API, where you're interacting with the actual model objects.